CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [Unreleased]
 
+### [3.3.1] - 2019-10-08
```

Comment:
Need to add link to diff and increment to v3.4.0.

Reply:
Link to diff?